assets/js/googlesitekit/datastore/site/info.js

```diff
@@ -83,6 +84,28 @@ export const actions = {
 			type: RECEIVE_PERMALINK_PARAM,
 		};
 	},
+
+	/**
+	 * Sets `siteKitAutoUpdatesEnabled` value; if set to `true` this will
+	 * enable auto-updates for Site Kit. Set to `false` to disable this
+	 * behaviour.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {boolean} siteKitAutoUpdatesEnabled Whether Site Kit auto-updates are enabled.
+	 * @return {Object} Redux-style action.
+	 */
+	setSiteKitAutoUpdatesEnabled( siteKitAutoUpdatesEnabled ) {
+		invariant(
+			siteKitAutoUpdatesEnabled,
+			'siteKitAutoUpdatesEnabled is required.'
+		);
```

Comment:
`invariant` will fail if called with any falsy value, so if this is called with `false` it will throw, when really we want this to error if it's called with no argument, or probably any non-boolean.